pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -182,6 +183,8 @@ let
       unzip
     ] ++ lib.optionals (format == "setuptools") [
       setuptoolsBuildHook
+    ] ++ lib.optionals (format == "build") [
```

Comment:
We should not be doing this. Format needs to disappear and was only introduced before PEP 517/518 was there.